modules/openapi/api/apis/dop/cmdb_backup_download.go

```diff
@@ -23,7 +23,7 @@ import (
 var CMDB_BACKUP_DOWNLOAD = apis.ApiSpec{
 	Path:          "/api/backup/<uuid>",
 	BackendPath:   "/api/files/<uuid>",
-	Host:          "dop.marathon.l4lb.thisdcos.directory:9527",
+	Host:          "core-services.marathon.l4lb.thisdcos.directory:9526",
```

Comment:
what backup means, and why just keep leave /api/backup/* and /api/files/* was deleted?

Reply:
Backup is a function provided by gittar.
Now it's not show on UI. So I will delete this openapi definition in this pr.

Reply:
`/api/files` just implemented on core-services, so I delete definitions which proxy to dop, remain core-services.